workers/reporter/src/main/kotlin/reporter/SourceCodeBundleReporter.kt

```diff
@@ -54,12 +57,38 @@ import org.slf4j.LoggerFactory
 const val SOURCE_BUNDLE_FILE_NAME = "source-bundle-archive.zip"
 const val SOURCE_BUNDLE_SUB_DIR = "source_code_bundle"
 
+data class SourceCodeBundleReporterConfig(
+    /**
+     * The license classifications to consider when including packages in the source code bundle.
+     */
+    val includedLicenseCategories: List<String>?,
+
+    /**
+     * The type of package to include in the source code bundle. Allowed values are "PROJECT" and "PACKAGE".
+     */
+    @OrtPluginOption(defaultValue = "PROJECT", aliases = ["packageType"])
```

Comment:
Unrelated as it was like that before, but: Shouldn't the default value rather be `PACKAGES` as it's more conservative because it does not expose the project's source code?

Reply:
I don't know, the source code of dependencies could also be internal. Maybe the best default would be to include both.